packages/components/organisms/f-header/src/components/Navigation.vue

```diff
@@ -320,6 +303,10 @@ export default {
         headerBackgroundTheme: {
             type: String,
             default: 'white'
+        },
+        showCountrySelector: {
+            type: Boolean,
+            default: false
```

Comment:
Do we need to have a default here, given that this component (Navigation) is only consumed internally to the Header module, and this prop is directly overriden by the prop we're passing the consuming component, or *it's* default if we don't set one?

Reply:
I think, if you don't make prop to be required it is a good practice to specify a default value to be safe if it is removed in future from the parent or you decide to extract this component.

Reply:
That's fair - I mention because we got caught out in one instance where the externally exposed prop was false by default, but the default in the internal component was true - the prop ended up resolving to true when the external prop was not set, essentially overriding the supposed default. Happy to accept as-is based on your reasoning.

Reply:
The [`vue/require-default-prop` lint rule](https://eslint.vuejs.org/rules/require-default-prop.html) will (or at least _should_) complain if you don't provide a default value as it's best practice to provide one, or alternatively, mark the prop as `required`.

Reply:
That `required` tip is good to know, @DamianMullins - given the issues we had before with the For You link I'd err on the side of required props over defaults for components internal to a module, but that might just be my preference.

Reply:
I'd only suggest using `required` when you really want users to explicitly set a value for the prop — it could get a little annoying having to set values for things you don't know/care about rather than having a sensible default value. 

I think setting a default value of false here and allowing the user to opt-in themselves seemed like a good way to go.